charts/metallb/templates/rbac.yaml

```diff
@@ -32,6 +32,9 @@ rules:
 - apiGroups: [""]
   resources: ["services", "endpoints", "nodes"]
   verbs: ["get", "list", "watch"]
+- apiGroups: ["discovery.k8s.io"]
+  resources: ["endpointslices"]
```

Comment:
I know it match current manifests, but isn't it needed for the controller also ?

Reply:
Nope, the same way endpoints are needed only for speakers.
The controller is started with ReadEndpoints = false which does not enable the listeners.
See https://github.com/metallb/metallb/blob/ebf611a825f04e62ad71f9eef1cea913ddab0611/controller/main.go#L154 vs 
https://github.com/metallb/metallb/blob/75e2dd302ec26ed979725f0f4bf7ab0718b7c0de/speaker/main.go#L134

Reply:
controller doesn't need to look at the endpoints, only at the service, thanks for clearing that
LGTM then

Reply:
Was there anything needed from me to move this pull request forward or does it look good as-is?

Reply:
i think we're good